packages/core-mobile/app/new/common/components/ScrollViewScreenTemplate.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  NavigationTitleHeader,
+  Text,
+  useKeyboardHeight
+} from '@avalabs/k2-alpine'
+import { useHeaderHeight } from '@react-navigation/elements'
+import { useFadingHeaderNavigation } from 'common/hooks/useFadingHeaderNavigation'
+import React, { useLayoutEffect, useRef, useState } from 'react'
+import { LayoutRectangle, View } from 'react-native'
+import {
+  KeyboardAwareScrollView,
+  KeyboardAwareScrollViewProps
+} from 'react-native-keyboard-controller'
+import Animated, {
+  interpolate,
+  useAnimatedStyle,
+  useSharedValue
+} from 'react-native-reanimated'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { BlurViewWithFallback } from './BlurViewWithFallback'
+import { KeyboardAvoidingView } from './KeyboardAvoidingView'
+import { LinearGradientBottomWrapper } from './LinearGradientBottomWrapper'
+import ScreenHeader from './ScreenHeader'
+
+interface FlatListScreenTemplateProps extends KeyboardAwareScrollViewProps {
+  title?: string
+  subtitle?: string
+  children: React.ReactNode
+  hasParent?: boolean
+  isModal?: boolean
+  disabled?: boolean
```

Comment:
```suggestion
  shouldAvoidKeyboard?: boolean
```
I think it’s better to base it on the enabled rather than the disabled.